gnocchi/transformation.py

```diff
@@ -0,0 +1,60 @@
+# -*- encoding: utf-8 -*-
```

Comment:
This seems to be used by the rest API for Carbonara, so I'd suggest moving it to `gnocchi.rest.transformation`.

Reply:
done